apps/web/src/components/SearchModal/ManageLists.tsx

```diff
@@ -209,7 +209,8 @@ function ManageLists({
           return (
             (chainId === ChainId.ETHEREUM && ETH_URLS.includes(listUrl)) ||
             (chainId === ChainId.BSC && BSC_URLS.includes(listUrl)) ||
-            (chainId === ChainId.POLYGON_ZKEVM && POLYGON_ZKEVM_URLS.includes(listUrl))
+            (chainId === ChainId.POLYGON_ZKEVM && POLYGON_ZKEVM_URLS.includes(listUrl)) ||
```

Comment:
Looks like getting harder to maintain. Need to refactor

Reply:
Yeah, better way should be to make it a multichain object